src/components/inputs/localized-rich-text-input/localized-rich-text-input.visualspec.js

```diff
@@ -0,0 +1,80 @@
+import { percySnapshot } from '@percy/puppeteer';
+import { getDocument, queries, wait } from 'pptr-testing-library';
+
+const { getByLabelText, getAllByLabelText, getByText } = queries;
+
+describe('LocalizedRichTextInput', () => {
+  const selectAllText = async input => {
+    // eslint-disable-next-line no-shadow
+    await page.evaluate(input => {
+      const range = document.createRange();
+      range.selectNodeContents(input);
+      const sel = window.getSelection();
+      sel.removeAllRanges();
+      sel.addRange(range);
+    }, input);
+  };
+
+  const getNumberOfTags = async tagName => {
+    // eslint-disable-next-line no-shadow
+    const numberOfTags = await page.evaluate(tagName => {
+      return document.querySelectorAll(tagName).length;
+    }, tagName);
+
+    return numberOfTags;
+  };
+
+  it('Default', async () => {
+    await page.goto(`${HOST}/localized-rich-text-input`);
+    await expect(page).toMatch('minimal');
+    await percySnapshot(page, 'LocalizedRichTextInput');
+  });
+
+  it('Interactive', async () => {
```

Comment:
I only put  a few tests here, I didn't see the point of copying all of the tests we already do in `RichTextInput`